cloudinfo/resourcemanager.go

```diff
@@ -22,3 +25,40 @@ func (infoSvc *CloudInfoService) GetResourceGroupIDByName(resourceGroupName stri
 
 	return "", fmt.Errorf("resource group with name %s not found", resourceGroupName)
 }
+
+func (infoSvc *CloudInfoService) CreateResourceGroup(name string) (*resourcemanagerv2.ResCreateResourceGroup, error) {
+	fmt.Println("Creating resource group: ", name)
+	resourceGroupOptions := infoSvc.resourceManagerService.NewCreateResourceGroupOptions()
+	resourceGroupOptions.SetName(name)
+	resourceGroup, _, err := infoSvc.resourceManagerService.CreateResourceGroup(resourceGroupOptions)
```

Comment:
I think this is better and update the return types appropriately
```suggestion
	return infoSvc.resourceManagerService.CreateResourceGroup(resourceGroupOptions)
}
```